### PR TITLE
Add caselaw data and extra sections to about page

### PR DESCRIPTION
**Pull request by Donna-H**
## Changes in this PR:
Add caselaw data and extra sections to about page
## Trello card / Rollbar error (etc)
https://trello.com/c/0beCW7Nv/1574-pui-update-about-page-so-there-is-a-caselaw-as-data-section
### Before
before-1
before-2
before-3

### After
after-1
after-2
after-3
after-4

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by Donna-H** [comment]
[PUI: Update 'about' page so there is a 'caselaw as data' section](https://trello.com/c/0beCW7Nv/1574-pui-update-about-page-so-there-is-a-caselaw-as-data-section)

File: ds_judgements_public_ui/templates/pages/about_this_service.html [hunk 1]
```diff
@@ -130,248 +139,322 @@ <h2 id="section-justice">Open justice</h2>
...
+    <a href="https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/openapi/public_api.yml">Case Law service
+      GitHub Project page.</a>.
+    </p>
+    <p>Example root feed URL:</p>
+    <p>
+      <a href="https://caselaw.nationalarchives.gov.uk/atom.xml">https://caselaw.nationalarchives.gov.uk/atom.xml</a>.
+    </p>
+    <p>URL Structure for Judgment Data:</p>
+    <p>
+      <a href="https://caselaw.nationalarchives.gov.uk/year/court/subdivision/id/data.xml">https://caselaw.nationalarchives.gov.uk/year/court/subdivision/id/data.xml</a>.
```
Comment (dragon-dxw) [inline ds_judgements_public_ui/templates/pages/about_this_service.html:188]: I'd make this not a link as it's not a real URL, unlike the atom feed.

File: ds_judgements_public_ui/templates/pages/about_this_service.html [hunk 2]
```diff
@@ -130,248 +139,322 @@ <h2 id="section-justice">Open justice</h2>
...
+    </p>
+    <p>In most cases the Neutral citation is how we uniquely identify the judgments and decisions.</p>
+    <h3>Parameters</h3>
+    <ul>
+      <li>
+        <code>order = updated :</code>  Sort by the last XML database update
```
Comment (dragon-dxw) [inline ds_judgements_public_ui/templates/pages/about_this_service.html:194]: I'd move the `:` outside the code block as it's not a part of the syntax.

File: ds_judgements_public_ui/templates/pages/about_this_service.html [hunk 3]
```diff
@@ -130,248 +139,322 @@ <h2 id="section-justice">Open justice</h2>
...
+      </li>
+      <li>
+        <code>order = date :</code> daSort by date on the judgment
```
Comment (dragon-dxw) [inline ds_judgements_public_ui/templates/pages/about_this_service.html:197]: as above

**Review by dragon-dxw — APPROVED** [review]

File: ds_judgements_public_ui/templates/pages/about_this_service.html (hunk 3 above)
Comment (Donna-H) [reply ds_judgements_public_ui/templates/pages/about_this_service.html]: Screenshot from 2024-01-12 15-00-37
Thank you updated @dragon-dxw

**Pull request merged**